streampark-plugin/streampark-jvm-profiler/src/main/java/org/apache/streampark/plugin/profiling/util/ClassAndMethod.java

```diff
@@ -53,10 +55,10 @@ public boolean equals(Object o) {
 
         ClassAndMethod that = (ClassAndMethod) o;
 
-        if (className != null ? !className.equals(that.className) : that.className != null) {
+        if (!Objects.equals(className, that.className)) {
             return false;
         }
-        return methodName != null ? methodName.equals(that.methodName) : that.methodName == null;
+        return Objects.equals(methodName, that.methodName);
```

Comment:
emmm, You know:
       `return Objects.equals(className, that.className) && Objects.equals(methodName, that.methodName);`